crypto/keyring/db_keybase.go

```diff
@@ -414,10 +414,11 @@ func (kb dbKeybase) writeInfo(name string, info Info) {
 	kb.db.SetSync(key, serializedInfo)
 
 	// store a pointer to the infokey by address for fast lookup
-	kb.db.SetSync(addrKey(info.GetAddress()), key)
+	kb.db.SetSync(kb.addrKeyByString(info.GetAddress()), key)
 }
 
-func addrKey(address types.AccAddress) []byte {
+// this is to be removed together with dbKeybase and the old Keybase interface
+func (kb dbKeybase) addrKeyByString(address types.AccAddress) []byte {
```

Comment:
So this has no business being a method on `dbKeybase`. Why not just move this to `crypto/keyring/keyring.go` where we have `addrKey` defined. We should probably rename both functions: `addrKey` and `addrHexKey`?

Reply:
> So this has no business being a method on dbKeybase

Nope, it doesn't. To be honest with you, I did it so that that I won't forget to remove it when the time to remove dbKeybase comes